oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSource.java

```diff
@@ -46,9 +51,23 @@ public OracleSource(OracleSourceConfig oracleSourceConfig) {
 
   @Override
   protected String createConnectionString(String host, Integer port, String database) {
+    if (OracleConstants.SERVICE_CONNECTION_TYPE.equals(this.connectionType)) {
+      return String.format(OracleConstants.ORACLE_CONNECTION_SERVICE_NAME_STRING_FORMAT, host, port, database);
+    }
+
     return String.format(OracleConstants.ORACLE_CONNECTION_STRING_FORMAT, host, port, database);
   }
 
+  @Path("getOracleSchema")
+  public Schema getOracleSchema(OracleSchemaRequest request,
```

Comment:
this way of getting schema is no longer supported and will fail to compile with 6.0.0. Whatever is set in configurePipeline is now what will be used. 

Don't have to fix in this PR, but let's be sure to fix it at some point in this repository. See https://github.com/cdapio/hydrator-plugins/blob/develop/database-plugins/src/main/java/io/cdap/plugin/db/batch/source/DBSource.java for how it is being done now.

Reply:
Ok